page_templates/style_guide.md

```diff
@@ -0,0 +1,55 @@
+
+## Guiding principles:
+
+-   **Accessibility**. Everyone who needs to should understand our documents.
+
+-   **Professional**. A consistent writing and formatting style will give a professional look to the documents.
+
+## Specific guidelines:
+
+**[ For anything not covered here, we default to the [Google developer style guide.](https://developers.google.com/style)  ]**
+
+1.  Check the spelling and grammar in your contributions. Most editors include a spell checker or have an available spell-checking plugin. You can also paste your text into a Google Doc or other document software for a more robust spelling and grammar check. We use American English spelling.
+
+2.  Avoid run-on sentences. Generally, keep sentences to less than 13 words longs.
+
+3.  When explaining technical steps, use a numbered step by step process. Don't list out every step in a single paragraph.
+
+4.  When explaining general concepts, paragraphs should be used. Yet paragraphs should not be over six sentences long.
+
+5.  Use the second person tense and describe actions to the user. (this is the same as active style or imperative style)
+
+> You should open Jina using python.
+> In your pod, you will find.
+
+6.  When using Jina terms, they should be Capitalized. Flow, Pod, JinaD. Anything found in the Jina glossary or Jina 101. Including Executor types (Evaluator, Segmenter, etc.)
+
+7. Don't capitalize Flow types (index, query). Don't capitalize the adjective or  `-tion`  form (i.e. evaluation, evaluating, segmenting, etc)
+
+8.  Use YAML, not yaml.
+
+9.  Do not use compound sentences, chains of clauses, and location-specific idioms, making text hard to understand and translate.
+
+10.  Avoid semicolons.
+
+11.  Don’t use abbreviations without explaining them first.
+
+12. When discussing a Jina component inside a code context (i.e. referring to CLI output or actual Python/YAML code) use  `Flow`,  `Pod`, etc
+
+13. For code/CLI output, capitalization should match the actual code or output
+
+14.  -   For section titles: Use sentence case
+		-   For top-level title: Use Title Case
+
+15. For code markup:
+- Use `<code>` in HTML or `` ` `` in Markdown to apply a monospace font and other styling to [code in text](https://developers.google.com/style/code-in-text), inline code, and user input.
+- Use code blocks, `<pre>` or ` ``` `, for [code samples](https://developers.google.com/style/code-samples) or other blocks of code.
+- Do not override or modify font styles inline.
+- Use code font to mark up code, such as class names, method names, HTTP status codes, console output, and placeholder variables.
+
+16. If you need to refer to installing Jina, use the following snippet:
+	   **Requirements**:
+	   - You have installed the latest stable release of Jina *X* according to the instructions found *here*. [Where *X* is replaced with the name of the Jina product you are talking about, such as Core/Hub. And *here* is replaced with an active link.]
```

Comment:
An active link to the install instruction doc. Not repo. Not website product section.

Pedantic is a pain until you get bitten by someone not doing literally the most obvious thing ;)

Reply:
Nope very good point! I tried to lay it out better?